salad/features/steps-browser-blur.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Make sure the focus and blur steps work
+
+
+    Scenario: 17. Focusing and blurring works
```

Comment:
Unclear why the scenarios are all numbered now.   Seems like it's going to add maintenance overhead keeping those numbers straight.  Rationale?